packages/flutter/lib/src/widgets/list_wheel_scroll_view.dart

```diff
@@ -20,6 +22,44 @@ import 'scroll_position.dart';
 import 'scroll_position_with_single_context.dart';
 import 'scrollable.dart';
 
+/// A delegate that supplies children with negative index by using builder
+/// callback.
+class NegativeIndexChildBuilderDelegate extends SliverChildBuilderDelegate {
+  /// Construct a delegate that lazily instantiate children using the given
+  /// builder callback.
+  ///
+  /// If childCount is null, then the list is infinite and the builder must not
+  /// return null. Otherwise, the builder will support children in index range
```

Comment:
Negative index doesn't necessarily imply infinite. The code below also allows builders to return null.

Reply:
Yes, but then it gets complicated because if there is either a lower/upper bound, we don't know about that value.
Anyway, I have fixed so that it should work fine even if the builder has an index lower or upper bound.
I will ping you after I commit the changes.